CIP-XXXX/README.md

```diff
@@ -0,0 +1,141 @@
+---
+CIP: ?
+Title: Orthogonal Certificates
+Status: Proposed
+Category: Meta
```

Comment:
```suggestion
Category: Ledger
```

Reply:
committing to ensure proper categorisation for meeting tomorrow.  @yHSJ - `Meta` is reserved for CIPs _about_ CIPs 🤓

Reply:
Thanks for the minor fixes @Ryun1!

Yes, @rphair, I obviously meant to change the category, but clearly forgot. I copied the header preamble from CIP-0001. Thanks for getting this committed

Reply:
yes @yHSJ I thought it was an artefact like that 😅